.gitignore

```diff
@@ -1,4 +1,5 @@
 vendor/
 *.pyc
 secrets.yaml
-*~
\ No newline at end of file
+*~
+*.db
```

Comment:
up to you?  personally, I have my developer environment ignores in a global `.gitignore` (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) since not everyone uses the same IDE's